client/webserver/site/src/js/orderutil.ts

```diff
@@ -59,21 +59,21 @@ export function isMarketBuy (ord: Order) {
 }
 
 /*
- * hasLiveMatches returns true if the order has matches that have not completed
+ * hasActiveMatches returns true if the order has matches that have not completed
  * settlement yet.
  */
-export function hasLiveMatches (order: Order) {
+export function hasActiveMatches (order: Order) {
   if (!order.matches) return false
   for (const match of order.matches) {
-    if (!match.revoked && match.status < MakerRedeemed) return true
+    if (match.active) return true
```

Comment:
The reason for status showing properly only on `/order/{id}` page on current **master** seems to have been the reliance on `!match.revoked` here in `function hasActiveMatches`. 

Best I can tell `function statusString` in JS must be same as `OrderReader.StatusString` in Golang (and it will be with the changes from this PR).

Reply:
https://github.com/decred/dcrdex/issues/2164

Need another solution for this.

Reply:
https://github.com/decred/dcrdex/pull/2140

Not clear about the root cause of that though.